### PR TITLE
TFM: Set MCUBOOT_IMAGE_NUMBER via Kconfig

### PR DESCRIPTION
**Pull request by oyvindronningstad**

### CODE REVIEW

**Review by microbuilder — APPROVED** [review]

**Review by ioannisg — APPROVED** [review]
Fine with me.
I'd phrase the help text slightly differently, though, indicating that
1 - means we do atomic Secure and Non-Secure FW updates
2 - means Secure and Non-Secure FW images are updated independently

I'd reflect that in the prompt text as well.

**Comment by oyvindronningstad** [comment]
> 
> 
> Fine with me.
> I'd phrase the help text slightly differently, though, indicating that
> 1 - means we do atomic Secure and Non-Secure FW updates
> 2 - means Secure and Non-Secure FW images are updated independently
> 
> I'd reflect that in the prompt text as well.

@ioannisg take a look now.

File: modules/trusted-firmware-m/CMakeLists.txt [hunk 1]
```diff
@@ -31,7 +31,7 @@
 #                        BUILD_PROFILE profile_small)
 function(trusted_firmware_build)
   set(options IPC REGRESSION)
-  set(oneValueArgs BINARY_DIR BOARD BL2 ISOLATION_LEVEL CMAKE_BUILD_TYPE BUILD_PROFILE)
+  set(oneValueArgs BINARY_DIR BOARD BL2 ISOLATION_LEVEL CMAKE_BUILD_TYPE BUILD_PROFILE MCUBOOT_IMAGE_NUMBER)
```
Comment (ioannisg) [inline modules/trusted-firmware-m/CMakeLists.txt:34]: see if checkpatch complains about the space at the end of the line :p

**Review by ioannisg — APPROVED** [review]

**Comment by ioannisg** [comment]
Need rebase now, @oyvindronningstad

**Review by tejlmand — APPROVED** [review]

**Pull request merged**